lib/.nvmrc

```diff
@@ -1 +1 @@
-10
+12
```

Comment:
burstjs is still compatible with NodeJS 10 - just put it to 12 due to husky pre-push, which triggers ng build